lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -384,10 +384,10 @@ public static function fileUploadFunction(array $params, $template)
             }
         }
 
-        $form = isset($params['form']) ? $params['form'] : null;
+        $form = isset($params['form']) ? ' form="' . $params['form'] . '"' : null;
```

Comment:
zgpdmie z tym co robisz niższej z `$form` to tu powinno być `''` zamiast `null`, bo inaczej będzie chyba php warning łączenia stringa z null.

Reply:
Nie będzie warninga ale proszę. Dla mnie **null** po prostu wygląda ładniej w kodzie niż ''.